services/emailservice.js

```diff
@@ -0,0 +1,87 @@
+
+/* 
+Rudimentary email service
+Currently creates emails for account creation, password changes, and error message if something else is attempted.
+
+TODO:
+- replace gmail with SMTP server
+- incorporate userID, so a client's username/password/email can be fetched with a GET request
+------ or will this already be done in some other piece of code?
+- make transporter password a function parameter, instead of just having the email password sit there in plain sight
+------ transporter's fields would have to be created in an outside function
+------ then, just do:
+------ let transporter = nodemailer.createTransport(setTransporterFields(password))
+
+NOTES:
+- got rid of "await" that was in front of transporter.sendMail(mail);
+------ while the code successfully sends emails now, info.messageId now becomes undefined. 
+------ does this matter? is there some other reason why it's better to have await?
+*/
+
+const nodemailer = require("nodemailer");
+
+async function main(){
+  // sets up info for email medium being used
+  let transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: '523pediatrics@gmail.com',
+      pass: 'monday1235'
+    }
+  });
+
+  // helper function: actually sends the email
+  function deliver(mail) {
+    // previously:
+    // let info = await transporter.sendMail(mail)
+    let info = transporter.sendMail(mail)
+    console.log("Message sent: %s", info.messageId);
+  }
+
+  // generate and send emails
+  // user, email, pass - information specific to the recipient of the email
+  // messageType       - specify what kind of message is being sent
+  // --- in the event messageType is invalid, admin account sends email to itself with inputted params
+  function sendNewEmail(messageType, user, email, pass) {
```

Comment:
We will probably want to split these into two different methods i.e. sendSignupEmail() and sendPasswordChangeEmail(), with both methods taking the user, email, and password, but not the messageType string.